datasets/imdb_labelled.txt

```diff
@@ -0,0 +1,1000 @@
+A very, very, very slow-moving, aimless movie about a distressed, drifting young man.  	0
+Not sure who was more lost - the flat characters or the audience, nearly half of whom walked out.  	0
```

Comment:
Labeled, not labelled, FYI.

Reply:
It is British VS US spelling. And it is original dataset's name, I did not change it. But I can remove extra "l" :)

Reply:
When in Rome ...

Reply:
Oh really? Hmmm. I never knew that. How strange. I guess if that's the original name we should keep it.